frontend/src/components/Search/CourseConflictIcon.tsx

```diff
@@ -47,26 +56,28 @@ function CourseConflictIcon({ course }: { readonly course: Listing }) {
     [course, data],
   );
 
+  const seasonMismatch = course.season_code !== CUR_SEASON;
```

Comment:
We can't do this because students in the fall season may still want to plan some spring classes; we should use `CUR_YEAR.includes(course.season_code)`